autogluon/training/buildspec.yml

```diff
@@ -28,7 +28,7 @@ images:
     image_size_baseline: 6825
```

Comment:
Just double check the CPU image size stay the same

Reply:
Yes, it roughly the same size.
Hence, keeping the baseline size similar.